modules/setup/setup.js

```diff
@@ -1,5 +1,5 @@
 const Docker = require('dockerode');
-var docker = new Docker({ socketPath: '/var/run/docker.sock' });
+var docker = new Docker({ socketPath: '/var/run/docker.sock', hosts: 'tcp://0.0.0.0:2376' });
```

Comment:
did you test on linux after these change of adding 'hosts'?

Reply:
yes, I tested on linux, that is working